forge-gui/res/editions/Murders at Karlov Manor.txt

```diff
@@ -101,6 +101,7 @@ ScryfallCode=MKM
 133 C Innocent Bystander @Warren Mahy
 134 U Knife @Tony Foti
 135 R Krenko, Baron of Tin Street @Brian Valeza
+136 R Krenko's Buzzcrusher @Joshua Raphael
```

Comment:
I'd say normally, don't worry about adding to the editions when scripting a card because it will be automatically added to the latest set if it's not in the edition and they get updated daily anyway, but may as well leave this in